test-cases/features/stop-compaction-ics.yaml

```diff
@@ -0,0 +1,14 @@
+test_duration: 60
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.large'
+
+backtrace_decoding: true
+
+user_prefix: 'stop-compaction-ics'
+append_scylla_yaml: |
+  enable_sstables_mc_format: true
+  enable_sstables_md_format: true
```

Comment:
This is only relevant for running sstables upgrade, I think we can safely remove it here.